websites/A/AniWorld/presence.ts

```diff
@@ -1,5 +1,5 @@
 const presence = new Presence({
-		clientId: "830517484472762408"
+		clientId: "969716001090437120"
```

Comment:
Have you asked the author about this?

Reply:
No, since this wasn't updated for quite a while... Also not really sure on how I would go about contacting the author.

Reply:
Seems like they left the sever. So, you could just take over then.